ui/app/templates/components/client-count/history.hbs

```diff
@@ -0,0 +1,91 @@
+        <EmptyState @title="No monthly history" @message="There is no data in the monthly history yet. We collect it at the end of each month, so your data will be available on the first of next month." />
+    {{else}}
+        <EmptyState @title="Data tracking is disabled" @message='Tracking is disabled, and no data is being collected. To turn it on, edit the configuration.'>
+        <p><LinkTo @route="vault.cluster.metrics.index" @query={{hash tab="config"}}>Go to configuration</LinkTo></p>
+        </EmptyState>
+    {{/if}}
+{{else}}
+<div class="box is-sideless is-fullwidth is-marginless is-bottomless">    
+    {{#if (eq @tab "current")}}
+        <h1 class="title is-4 has-bottom-margin-s">Current month</h1>
+        <p class="has-bottom-margin-s">The below data is for the current month starting from the first day. For historical data, see the monthly history tab.</p>
+        {{#if (eq @model.config.enabled 'Off')}}
+            <EmptyState @title="Tracking is disabled" @message="Tracking is disabled and data is not being collected. To turn it on edit the configuration.">
+                <LinkTo @route="vault.cluster.metrics.edit">Go to configuration</LinkTo>
+            </EmptyState>
+        {{/if}}
+    {{else}}
+        {{#if (eq @model.config.enabled 'Off')}}
+             <AlertBanner
+                data-test-tracking-disabled
+                @type="warning"
+                @title="Tracking is disabled"
+            >
+            Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need to <LinkTo @route="vault.cluster.metrics.edit">edit the configuration</LinkTo> to enable tracking again.
+            </AlertBanner>
+        {{/if}}
+        <h1 class="title is-4 has-bottom-margin-s">Monthly history</h1>
+        <p class="has-bottom-margin-s">This data is presented by full month. If there is data missing, it's possible that tracking was turned off at the time. Vault will only show data for contiguous blocks of time during which tracking was on.</p>
+        <PricingMetricsDates
+        @queryStart={{@model.queryStart}}
+        @queryEnd={{@model.queryEnd}}
+        @resultStart={{@model.activity.startTime}}
+        @resultEnd={{@model.activity.endTime}}
+        @defaultSpan={{or @model.config.defaultReportMonths 12}}
+        @retentionMonths={{@model.config.retentionMonths}}
+        />
+    {{/if}}
+    
+    {{#unless @model.activity.total}}
+        {{#if (eq @tab "current")}}
+            {{#if (eq @model.config.enabled 'On')}}
+                <EmptyState @title="No data received" @message="Tracking is turned on and Vault is gathering data. It should appear here within 30 minutes" />
+            {{/if}}
+        {{else}}
+                <EmptyState @title="No data received" @message="No data exists for that query period. Try searching again." />
+        {{/if}}
+    {{else}}
+        <div class="card has-bottom-margin-m">
```

Comment:
Since there's talk of removing Bulma, do we want to start building templates without using helpers like `card` and `columns`? Just something to consider as we move forward, not sure if it's even worth refactoring now without an alternative in place. But might be worth chatting about as a team so we all have a consistent way of styling until removing bulma.

Reply:
Yeah I think its better if we first work on alternatives and then iteratively replace bulma helpers.
I'm not sure if we are going with our own components, use some other library or waiting for design system team. 
Let's discuss with @chelshaw when she is back from break.